docusaurus/docs-internals/editor-controls/data-controls/inputText.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 2
+---
+
+# InputText
+
+Brizy's inputText control provides a straightforward input field for text entry, enabling users to effortlessly input and edit text, ensuring a seamless and user-friendly experience.
```

Comment:
i think we can remove part with: "ensuring a seamless and user-friendly experience"